### PR TITLE
Update Base.java alongside todo.txt

### LINKED ISSUE #886: Remove dependency on `todo.txt` and automate `app/src/Base.java`

### PR DESCRIPTION
**Pull request by Stefterv**
Fixes #886

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -93,9 +93,10 @@ jobs:
           architecture: ${{ matrix.arch }}
       - name: Setup Ant
         uses: cedx/setup-ant@v3
-      - name: Write to todo.txt
+      - name: Write to `todo.txt` and update `Base.java`
```
Comment (SableRaf) [inline .github/workflows/release.yml]: I'd suggest a clearer name like "Write build_number (revision) to `todo.txt` and `Base.java`"

**Comment by SableRaf** [comment]
@Stefterv Are `build_number` and `revision` different things?

**Comment by Stefterv** [comment]
No there is a build number and a version, but the base.java is bit confusion with revision and version name which are both historically the same value

**Review by SableRaf — APPROVED** [review]

**Comment by SableRaf** [comment]
@Stefterv Thanks for clarifying! Looks good to me 👍

**Pull request merged**